.travis.yml

```diff
@@ -2,6 +2,8 @@ sudo: false
 language: python
 python:
   - 2.7
+  - 3.4
+  - 3.5
```

Comment:
I chose these 2 since they're the only versions that Django 1.9 supports.

https://docs.djangoproject.com/en/1.9/releases/1.9/